exercises/03-Interacting_with_services.md

```diff
@@ -37,20 +37,20 @@ The simplest API for an exposed service is simply returning something useful in
   2. Compare to an extension whose service URIs do not have the trailing slash: <code>http://**localhost**/fcrepo/rest/apix/extensions/fits</code>
 
-3. Continuing on from step 1, follow the service endpoint URI for the xml extension in your browser: <code>http://**localhost**/services/apix/extensions/svc:xmlmetadata/</code>
+3. Continuing on from step 1, follow the service endpoint URI for the xml extension in your browser: <code>http://**localhost**/services/apix/extensions/demo:rdfvis/</code>
 You'll see a simple html document that presents two choices.  In your browser, view its source.  As you can probably guess, this is just a static html resource; being able to use relative URIs lets us get away with that easily.
 
-4. Follow the link to the DC format xml.  The extension applies an XSLT to select any rdf properties that correspond to Dublin Core elements.  If there are none in the resource, as is the case in the example, then the resulting document is largely empty; the same is true for MODS.  For our purposes, we're more interested in looking at its URI:
-<code>http://**localhost**/services/apix/extensions/svc:xmlmetadata/dc</code>
-  * The underlying service that implements this extension happens to be implemented in Java using Apache Camel.  [These lines of code] (https://github.com/birkland/repository-extension-services/blob/apix-demo/acrepo-exts-serialize-xml/src/main/java/edu/amherst/acdc/exts/serialize/xml/EventRouter.java#L38-L49) implement the routing logic with respect to the incoming service request.
-  * In exercise 4, we will explore extension deployment and registering services in a little more detail.  Suffice it to say, the underlying implementation of the xml service in this demo is `http://acrepo:9104/xml`.  A request to API-X <code>http://**localhost**/services/apix/extensions/svc:xmlmetadata/dc</code> gets forwarded via the API-X to `http://acrepo:9104/xml/dc`.  Read the [execution engine](https://github.com/fcrepo4-labs/fcrepo-api-x/blob/master/src/site/markdown/execution-and-routing.md#generic-endpoint-proxy) documentation to get a greater understanding of the interaction between API-X and the underlying service.
+4. Follow the link to "View as image".  This extension renders the RDF in the repository resource as an image.  For our purposes, we're more interested in looking at its URI:
+<code>http://**localhost**/services/apix/extensions/demo:rdfvis/image</code>
+  * The underlying service that implements this extension happens to be implemented in PHP.  [These lines of code] (https://github.com/birkland/fcrepo-api-x-demo/blob/acrepo-1.1.0/extensions/rdfvis/1.0.0/www/index.php#L15-L30) implement the routing logic with respect to the incoming service request.  If no path is present (or it doesn't match), the html present after this code block will be displayed.
```

Comment:
For some reason the
> [These lines of code] 

 link isn't rendering properly in markdown.  Maybe a space or something?